proto/faas/v1/faas.proto

```diff
@@ -0,0 +1,130 @@
+syntax = "proto3";
+package nitric.faas.v1;
+
+// protoc plugin options for code generation
+option go_package = "nitric/v1;v1";
+option java_package = "io.nitric.proto.faas.v1";
+option java_multiple_files = true;
+option java_outer_classname = "NitricFaas";
+option php_namespace = "Nitric\\Proto\\Faas\\V1";
+option csharp_namespace = "Nitric.Proto.Faas.v1";
+
+// Service for management of event topics
+service Faas {
+  // Return a list of existing topics in the provider environment
+  rpc TriggerStream (stream ClientMessage) returns (stream ServerMessage);
+}
+
+// Messages the client is able to send to the server
+message ClientMessage {
+  // Client message ID, used to pair requests/responses
+  string id = 1;
+
+  oneof content {
+    // Client initialisation request
+    InitRequest init_request = 2;
+
+    // Client responsding with result of
+    // a trigger
+    TriggerResponse trigger_response = 3; 
+  }
+}
+
+// Messages the server is able to send to the client
+message ServerMessage {
+  // Server message ID, used to pair requests/responses
+  string id = 1;
+
+  oneof content {
+    // Server responding
+    // with client configuration details to an
+    // InitRequest
+    InitResponse init_response = 2;
+
+    // Server requesting client to
+    // process a trigger
+    TriggerRequest trigger_request = 3;
+  }
+}
+
+// Placeholder message
+message InitRequest {}
+
+// Placeholder message
+message InitResponse {}
+
+
+// The membrane has a trigger for this function to handle
```

Comment:
Membrane has a trigger for the *client* to handle

Reply:
fixed.